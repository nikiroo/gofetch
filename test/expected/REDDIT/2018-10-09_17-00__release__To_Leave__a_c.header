0[release] To Leave, a challenging puzzle-platformer with an emotionally gripping narrative, released on Steam today with day 1 Linux support. 20% off during its first week.	null/REDDIT/2018-10-09_17-00__release__To_Leave__a_c		70
i  [linux_gaming] 2018-10-09_17-00
i
