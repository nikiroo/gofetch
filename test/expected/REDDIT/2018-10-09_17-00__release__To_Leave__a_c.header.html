<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h2><a href='2018-10-09_17-00__release__To_Leave__a_c.html'>[release] To Leave, a challenging puzzle-platformer with an emotionally gripping narrative, released on Steam today with day 1 Linux support. 20% off during its first week.</a></h2>
	<div class='details'>([linux_gaming] 2018-10-09_17-00)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		
	</div>
<hr/>
</div>
</body>
